zerver/test_hooks.py

```diff
@@ -834,3 +835,31 @@ def test_unknown_message_type(self):
             msg.content,
             'Unknown pagerduty message\n``` py\n{u\'type\': u\'foo\'}\n```'
         )
+
+class TravisHookTests(AuthedTestCase):
+    def test_travis_message(self):
+        """
+        Build notifications are generated by Travis after build completes.
+
+        The subject describes the repo and Stash "project". The
+        content describes the commits pushed.
+        """
+        email = "hamlet@zulip.com"
+        api_key = self.get_api_key(email)
+        body =urllib.urlencode({'payload': self.fixture_data("travis", "build", file_type="json") })
```

Comment:
Missing space after the equals sign and also there should be no space between ) and } near the end.